docs/source/io_formats/nuclear_data.rst

```diff
@@ -16,23 +16,40 @@ Incident Neutron Data
              - **metastable** (*int*) -- Metastable state (0=ground, 1=first
                excited, etc.)
              - **atomic_weight_ratio** (*double*) -- Mass in units of neutron masses
-             - **temperature** (*double*) -- Temperature in MeV
+             - **kTs** (*double[]*) -- Temperatures (in MeV) contained in the library
              - **n_reaction** (*int*) -- Number of reactions
 
 :Datasets: - **energy** (*double[]*) -- Energy points at which cross sections are tabulated
 
+**/<nuclide name>/kTs/**
+
+<TTTK> is the temperature in Kelvin, rounded to the nearest integer, of the
+temperature-dependent data set.  For example, the data set corresponding to
+300 Kelvin would be located at `300K`.
+
+:Datasets: - **<TTTK>** (*double[]*) -- kT values (in MeV) for each Temperature
```

Comment:
I would write this as `<TTT>K` to be clear that only TTT is the parameter. Also, it should be `double`, not `double[]`.